developer_manual/app/info.rst

```diff
@@ -117,6 +117,13 @@ ownCloud allows to specify four kind of "types". Currently supported "types":
 
 * **logging**: apps which implement a logging system
 
+* **prevent_group_restriction**: apps which can not be enabled for specific groups (e.g. notifications app).
```

Comment:
No, we have an option that allows restricting an app, so it is only enabled for some groups:
![apps_-_owncloud_-_2016-01-14_16 20 46](https://cloud.githubusercontent.com/assets/213943/12328246/d5ddcaa0-bada-11e5-8175-df5930519d18.png)

However, there are apps where that makes no sense: activities, notifications, provisioning api, etc.

Currently they all use "filesystem" to avoid it, but that means they are loaded in places and at a stage where they don't need to be loaded. So in order to provide this feature to non-filesystem apps, we introduce a new app type.